### ISSUE TITLE
The effect of Mix3D data augmentation.

### ISSUE THREAD

**Issue by RongkunYang** [Pointcept/PointTransformerV3#35]
Dear authors, 
    thank you for share your great works.
    When I read the code, I found this code use the Mix3D data augmentation, this augmentation achieves good performance on MinkowskiNet backbone for ScanNet in the Mix3D paper.
    So I'd like to ask how much the Mix3D data augmentation affect the performance of Point Transformer v3 backbone.

Thank you.

**Comment by RongkunYang** [comment]
I also would like to know whether Mix3D is effective in PointGroup instance segmentation, have you tried this experiments on PointGroup, thank you very much.

**Comment by Gofinge** [comment]
> how much the Mix3D data augmentation affect the performance of Point Transformer v3

In my experiments, Mix3D can not directly increase mIoU results for 3D backbones, but it can make validation loss better, which means doing something good to prevent overfitting. So, we include it in our recipe.

>  whether Mix3D is effective in PointGroup instance segmentation, have you tried this experiments on PointGroup

I haven't tried this, and I think it is not trivial to implement it.
